package.json

```diff
@@ -67,6 +69,7 @@
     "lint": "eslint --fix bin/marked .",
     "build:reset": "git checkout upstream/master lib/marked.js lib/marked.esm.js marked.min.js",
     "build": "npm run rollup && npm run minify",
+    "build:docs": "node build-docs.js",
```

Comment:
Does this have to be called manually any time the docs are changed? Should we run this in the [Build step](https://github.com/markedjs/marked/blob/master/.github/workflows/tests.yml#L40) on merge?

Reply:
We don't commit the docs output so it doesn't need to run unless you want to see the docs output.

Would you prefer I rename this script to `vercel-build` instead of `build:docs` so its clear its for Vercel?

Reply:
How does vercel know to run `npm build:docs` to get the output? and when does it run (on every PR, on every commit to master)?

Reply:
The build command is configured here: https://vercel.com/markedjs/markedjs/settings

It runs every `git push` regardless of the branch. The `master` branch gets the production domain, https://marked.js.org, and PRs get the preview domains such as https://markedjs-git-static-docs.markedjs.vercel.app